library/src/engine.cxx

```diff
@@ -213,7 +213,7 @@ interactor& engine::getInteractor()
 }
 
 //----------------------------------------------------------------------------
-void engine::loadPlugin(const std::string& pathOrName, const std::vector<std::string>& searchPaths)
+void engine::loadPlugin(std::string_view pathOrName, const std::vector<fs::path>& searchPaths)
```

Comment:
`pathOrName` is used many times to create a string, I think keeping it const ref might be better?